workflows-samples/README.md

```diff
@@ -0,0 +1,21 @@
+# Batch Workflows samples
+This directory contains samples of Batch related GCP [workflows](https://cloud.google.com/workflows)
+
+## Common prerequisites for all samples
+* You have a [Google Cloud project](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+* You have enabled the [Batch API](https://console.cloud.google.com/batch) and understand the [basic Batch concepts](https://cloud.google.com/batch/docs/get-started#product-overview).
+* You have the permissions required to [create Batch jobs](https://cloud.google.com/batch/docs/create-run-basic-job).
+* You should know about how to create and deploy GCP Workflows, more details in [Workflows Overview](https://cloud.google.com/workflows/docs/overview).
+* To grant the necessary IAM roles to the Workflows runner by following [Workflows IAM control guide](https://cloud.google.com/workflows/docs/use-iam-for-access).
+
+## Samples
+### export-to-bigquery-delete-batch-jobs
```

Comment:
Need to enable BQ API for this sample?

Reply:
That's a good point, I forgot about it since I was using the `owner` role for testing, I can make a follow-up commit to add that part.